ai-ml/llm-multiple-gpus/falcon-40b/text-generation-inference.yaml

```diff
@@ -51,7 +51,7 @@ spec:
         volumeMounts:
           - mountPath: /dev/shm
             name: dshm
-          - mountPath: /data
+          - mountPath: /tmp
```

Comment:
Since this mounthPath is tightly coupled with the container image used. Can we add some comment or details around why it needs to be exactly `/tmp`. This would avoid someone changing it later resulting into things not working ?